samples/Gallery/Pages/ThemeAwarePage.fs

```diff
@@ -62,26 +59,22 @@ module ThemeAwarePage =
         )
 
     let view () =
-        Component(program) {
-            let! model = Mvu.State
+        Component("ThemeAwarePage") {
+            let! model = Context.Mvu program
+            let! theme = Context.Environment(EnvironmentKeys.Theme)
 
             VStack(spacing = 15.) {
-                let requestedThemeVariant =
-                    FabApplication.Current.RequestedThemeVariant
-                    |> Option.ofObj
-                    |> Option.defaultValue ThemeVariant.Default
-
-                let actualThemeVariant =
-                    FabApplication.Current.ActualThemeVariant
-                    |> Option.ofObj
-                    |> Option.defaultValue ThemeVariant.Default
-
-                TextBlock($"Requested theme variant is: {requestedThemeVariant.ToString()}")
-                TextBlock($"Actual theme variant is: {actualThemeVariant.ToString()}")
+                TextBlock($"Actual theme variant is: {theme.ToString()}")
                 TextBlock($"ScopedTheme is: {model.ScopeTheme.ToString()}")
 
+                let color =
+                    if theme = ThemeVariant.Light then
+                        Colors.Red
+                    else
+                        Colors.Green
+
                 TextBlock("I'm a text that is theme aware.")
-                    .foreground(SolidColorBrush(ThemeAware.With(Colors.Red, Colors.Green)))
+                    .foreground(SolidColorBrush(color))
```

Comment:
I see... My question was regarding how the `theme` is used here. This `if` statement was hidden by the `ThemeAware.With` helper before. Will there be a helper to replace it?

Reply:
Hum I guess we can add it back. It would be 2 ways of doing the same. But I think these can co-exist with not problem.

Reply:
Thanks, that'd make widgets with themed colors and the migration to pre7 easier.